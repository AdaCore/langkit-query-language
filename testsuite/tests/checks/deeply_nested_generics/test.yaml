driver: 'checker'
rule_name: 'Deeply_Nested_Generics'
project: 'prj.gpr'
rule_arguments:
    deeply_nested_generics.n: 1
