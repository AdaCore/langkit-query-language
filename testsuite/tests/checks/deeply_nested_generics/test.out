deeply_nested_generics.ads:8:25: error: deeply nested generic
8 |       generic procedure P;
  |                         ^ 

deeply_nested_generics.ads:12:15: error: deeply nested generic
12 |       package P_G_1 is --  FLAG (depth=1)
   |               ^^^^^ 

deeply_nested_generics.ads:14:18: error: deeply nested generic
14 |          package P_G_2 is -- FLAG (depth=2)
   |                  ^^^^^ 

